src/systemctl/systemctl.c

```diff
@@ -75,7 +75,7 @@ bool arg_no_wtmp = false;
 bool arg_no_sync = false;
 bool arg_no_wall = false;
 bool arg_no_reload = false;
-bool arg_value = false;
```

Comment:
same question about `arg_all` here

Reply:
Same for `systemctl`. `arg_all` is used at various places.